### PR TITLE
docs(async): fix doc of `retry.ts` #4129

### LINKED ISSUE #4129: Doc of `async/retry.ts` is rendered badly
**Describe the bug**

Visit https://deno.land/std@0.211.0/async/retry.ts?s=retry

The doc of this module is rendered kinda badly

Pasted_Image_2024_01_08_18_03

**Expected behavior**

It should be rendered prettily, just like its doc of v0.208.0.
https://deno.land/std@0.208.0/async/retry.ts?s=retry

**Environment**

N/A

### PR DESCRIPTION
**Pull request by kyu08**
Resolves #4129

## What I tested
1. Run `deno doc --html --name=std/async async/mod.ts`
2. Open `docs/index.html`

I confirmed that the rendering issue reported in #4129 has been resolved.
image

### CODE REVIEW

File: async/retry.ts [hunk 1]
```diff
@@ -74,10 +74,8 @@ const defaultRetryOptions: Required<RetryOptions> = {
  * Retries as long as the given function throws. If the attempts are exhausted,
  * throws a {@linkcode RetryError} with `cause` set to the inner exception.
  *
- * The backoff is calculated by multiplying `minTimeout` with `multiplier` to
- * the power of the current attempt counter (starting at 0 up to `maxAttempts
- * - 1`). It is capped at `maxTimeout` however. How long the actual delay is,
- * depends on `jitter`.
+ * The backoff is calculated by multiplying `minTimeout` with `multiplier` to the power of the current attempt counter (starting at 0 up to `maxAttempts - 1`). It is capped at `maxTimeout` however.
+ * How long the actual delay is, depends on `jitter`.
```
Comment (kyu08) [inline async/retry.ts:78]: I fixes based on this. https://github.com/denoland/deno_std/blob/e0ef24091e87f84d44d495d432d611625b281249/async/retry.ts#L41

**Comment by kyu08** [comment]
Hi, @magurotuna @kt3k 
I have one question.
Is there a way to check the documents are rendered properly in my machine or some preview environments?

**Comment by kt3k** [comment]
@kyu08 Try this command: `deno doc --html --name=std/async async/mod.ts` and open `docs/index.html`

**Comment by kyu08** [comment]
@kt3k 
Thank you for letting me know that useful command! It worked well!

Also, I added a test result to description and make this PR open.
Could you review it?

**Comment by magurotuna** [comment]
Thanks for the very quick fix!

**Review by kt3k — APPROVED** [review]
LGTM. Thanks!

**Pull request merged**